src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayPrice = 40;
+  const sevenDays = 7;
+  const threeDays = 3;
```

Comment:
```suggestion
  const longTerm = 7;
  const shortTerm = 3;
```
немає сенсу в назвах типу sevenDays. Змінні створюються для того, щоб містити в собі якусь конкретну інформацію. В даному випадку нам потрібно містити кількість днів, які вважаються коротким чи довгим терміном. І якби ми в подальшому захотіли б змінити якись термін, нам треба було б просто поміняти значення в одному місці